### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by yaneshtyagi**

### CODE REVIEW

**Comment by yaneshtyagi** [comment]
@yanshityagi Please read and review this PR. The contents are written in a very easy non-technical language that you should be able to comprehend. It's plain English text, not any kind of code. 

No hurry. You can do this whenever you get time in next couple of days.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
```
Comment (yanshityagi) [inline CODE_OF_CONDUCT.md:62]: enforcement at what??

**Comment by yanshityagi** [comment]
I have reviewed but there is a doubt (that we discussed).
I have left a comment on it.

Regards,
Yanshi Tyagi

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (yaneshtyagi) [reply CODE_OF_CONDUCT.md]: The sentence was incomplete. It should be `enforcement at Rovisor`. Corrected and pushed the updates.

**Comment by yaneshtyagi** [comment]
@yanshityagi I made the corrections based on your comment. Please review again.

**Review by yanshityagi — APPROVED** [review]

**Comment by yanshityagi** [comment]
Approved.

Regards,
Yanshi Tyagi

**Pull request merged**